modules/mri_violations/php/NDB_Menu_Filter_Form_mri_violations.class.inc

```diff
@@ -147,6 +147,13 @@ class NDB_Menu_Filter_Form_mri_violations extends NDB_Menu_Filter_Form
         else{
             $useProjects = true;
         }
+        // Get sites
+        $sitesCond = '';
+        if (!$user->hasPermission('access_all_profiles') 
+            && !is_null($user->getData('CenterID'))
+        ) {
+            $sitesCond = " AND p.CenterID = '".$user->getData('CenterID')."' ";
```

Comment:
Remark: p.CenterID is a numeric column and not text. The ' characters should be removed.
But [MySQL](http://dev.mysql.com/doc/refman/5.7/en/type-conversion.html) is taking care of it.